### PR TITLE
Implemented swap even and odd bits program in C language inside the B…

### PR DESCRIPTION
**Pull request by srishti-13**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Fixed Issue number #4282 Swap even and odd bits

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by srishti-13** [comment]
Fixes issue number #4282

File: C/Bit--Manipulation/Swap_even_and_odd_bits.c [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
```
Comment (atarax665) [inline C/Bit--Manipulation/Swap_even_and_odd_bits.c:1]: Add a description of the program.

File: C/Bit--Manipulation/Swap_even_and_odd_bits.c [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    */
+
+    /*A in binary is 1010 . So, by performing bitwise & operation, we will get all the even bits of number x since, 1 is 
+   present at even positions. (Masking) */
+
+    unsigned int oddBits = x & 0x55555555; //5 in decimal is 0101. So, by performing bitwise & operation, we will get
+                                           // all the odd bits of number x since 1 is present at odd positions.
+                                           //(Masking)
+
+    evenBits >>= 1; // Right shifting even bits
```
Comment (atarax665) [inline C/Bit--Manipulation/Swap_even_and_odd_bits.c:32]: Shift the comment above the line of code.

File: C/Bit--Manipulation/Swap_even_and_odd_bits.c [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    unsigned int x;
+    printf("Enter the number whose bits need to be swapped:-\n");
+    scanf("%u", &x);
+    printf("%u ", swapBits(x));
+    return 0;
+}
+
+/* Time Complexity:- O(1)
+   Space Complexity:- O(1)
+*/
```
Comment (atarax665) [inline C/Bit--Manipulation/Swap_even_and_odd_bits.c:50]: Add sample I/O.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Update appropriate README.md.

**Comment by srishti-13** [comment]
> Update appropriate README.md.

Yes, I have done it. Now, you can review my code again.

File: C/Bit--Manipulation/Swap_even_and_odd_bits.c (hunk 1 above)
Comment (srishti-13) [reply C/Bit--Manipulation/Swap_even_and_odd_bits.c:1]: Yes, I have done it. Now, you can review my code again.

File: C/Bit--Manipulation/Swap_even_and_odd_bits.c (hunk 2 above)
Comment (srishti-13) [reply C/Bit--Manipulation/Swap_even_and_odd_bits.c:32]: Yes, I have done it. Now, you can review my code again.

File: C/Bit--Manipulation/Swap_even_and_odd_bits.c (hunk 3 above)
Comment (srishti-13) [reply C/Bit--Manipulation/Swap_even_and_odd_bits.c:50]: Yes, I have done it. Now, you can review my code again.

File: C/Bit--Manipulation/Swap_even_and_odd_bits.c (hunk 1 above)
Comment (HarshCasper) [reply C/Bit--Manipulation/Swap_even_and_odd_bits.c:1]: No. You have not resolved it yet

File: C/Bit--Manipulation/Swap_even_and_odd_bits.c (hunk 2 above)
Comment (HarshCasper) [reply C/Bit--Manipulation/Swap_even_and_odd_bits.c:32]: No. You have not resolved it yet

File: C/Bit--Manipulation/Swap_even_and_odd_bits.c (hunk 3 above)
Comment (HarshCasper) [reply C/Bit--Manipulation/Swap_even_and_odd_bits.c:50]: No. You have not resolved it yet

**Comment by HarshCasper** [comment]
@srishti-13 You have not resolved any of the changes requested. Please make them for any further review.

File: C/Bit--Manipulation/Swap_even_and_odd_bits.c (hunk 1 above)
Comment (srishti-13) [reply C/Bit--Manipulation/Swap_even_and_odd_bits.c:1]: > No. You have not resolved it yet

But, I have resolved it and in the commits section, it is showing. Could you tell me any reason why it is not showing the changes?

**Review by ankitaggarwal23 — COMMENTED** [review]
Code plagiarized from [here](https://www.geeksforgeeks.org/swap-all-odd-and-even-bits/).

**Pull request closed without merging**